site/app/views/PollView.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace app\views;
+
+use app\models\User;
+use app\libraries\FileUtils;
+
+class PollView extends AbstractView {
+
+    public function showPollsInstructor($todays_polls, $older_polls, $future_polls) {
+        $this->core->getOutput()->addBreadcrumb("Polls");
+        $this->core->getOutput()->addInternalCss('polls.css');
+        $this->core->getOutput()->addInternalJs('polls-dropdown.js');
+        $this->core->getOutput()->enableMobileViewport();
+        return $this->core->getOutput()->renderTwigTemplate("polls/AllPollsPageInstructor.twig", [
+            'csrf_token' => $this->core->getCsrfToken(),
+            'base_url' => $this->core->buildCourseUrl() . '/polls',
+            'todays_polls' => $todays_polls,
+            'older_polls' => $older_polls,
+            'future_polls' => $future_polls
+          ]);
+    }
+
+    public function showPollsStudent($todays_polls, $older_polls) {
+        $this->core->getOutput()->addBreadcrumb("Polls");
+        $this->core->getOutput()->addInternalCss('polls.css');
+        $this->core->getOutput()->addInternalJs('polls-dropdown.js');
+        $this->core->getOutput()->enableMobileViewport();
+        return $this->core->getOutput()->renderTwigTemplate("polls/AllPollsPageStudent.twig", [
+            'csrf_token' => $this->core->getCsrfToken(),
+            'base_url' => $this->core->buildCourseUrl() . '/polls',
+            'todays_polls' => $todays_polls,
+            'older_polls' => $older_polls,
+            'user_id' => $this->core->getUser()->getId(),
+          ]);
+    }
+
+    public function showNewPollPage() {
+        $this->core->getOutput()->addBreadcrumb("Polls", $this->core->buildCourseUrl(["polls"]));
+        $this->core->getOutput()->addBreadcrumb("New Poll");
+        $this->core->getOutput()->addInternalCss('polls.css');
```

Comment:
It would be nice if you added `$this->core->getOutput()->enableMobileViewport();` to make this work on mobile. I think the page should be fine css wise as is on mobile so hopefully this is the only change you need